batch/batch/front_end/front_end.py

```diff
@@ -427,20 +469,19 @@ async def _get_full_job_status(app, record):
 
 @routes.get('/api/v1alpha/batches/{batch_id}/jobs/{job_id}/log')
 @prom_async_time(REQUEST_TIME_GET_JOB_LOG)
-@rest_authenticated_users_only
-async def get_job_log(request, userdata):  # pylint: disable=R1710
+@rest_billing_project_users_only
+async def get_job_log(request, userdata):  # pylint: disable=unused-argument
     batch_id = int(request.match_info['batch_id'])
     job_id = int(request.match_info['job_id'])
-    user = userdata['username']
-    job_log = await _get_job_log(request.app, batch_id, job_id, user)
+    job_log = await _get_job_log(request.app, batch_id, job_id)
     return web.json_response(job_log)
 
 
-async def _query_batches(request, user):
+async def _query_batches(request):
     db = request.app['db']
 
-    where_conditions = ['user = %s', 'NOT deleted']
-    where_args = [user]
+    where_conditions = ['NOT deleted']
```

Comment:
This method appears to now have no restrictions at all?

Reply:
Again, see comment above. We have to do something different for query_batches or queries that don't have a specific batch idea like query_billing (which I already implemented previously).